clog/clog.go

```diff
@@ -79,12 +89,39 @@ func toOutputAtLevel(outputHandler log.Handler, lvl log.Lvl) {
 	setRootHandler(h)
 }
 
+// CallerInfoHandler returns a Handler that, at the Debug, Warn and Error
+// levels, adds the file and line number of the calling function to the context
+// with key "caller". At the Crit levels it instead adds a stack trace to the
+// context with key "stack". The stack trace is formatted as a space separated
+// list of call sites inside matching []'s. The most recent call site is listed
+// first.
+//
+// taken from https://github.com/sb10/l15h/blob/master/l15h.go#L143
```

Comment:
Move this kind of copyright information reference to the copyright block at the top of the file; it shouldn't be in the public documentation for the function.